### PR TITLE
Fingerprint revamp

### PR DESCRIPTION
**Pull request by maryann-macrometa**
First stage of Fingerprint revamp. Still testing the client so still a work in progress.

### CODE REVIEW

File: docs/photoniq/fingerprinting/get-started-fingerprinting.md [hunk 1]
```diff
@@ -52,9 +62,11 @@ In this example, the JavaScript is loaded at page load time, and the function to
 </body>
 ```
 
-## Fingerprint on Button Click
+</TabItem>
 
-In this example, the JavaScript is initially loaded at page load time, and a function is created that is tied to a button. Once the button is clicked, a call is made to the fingerprint server, which returns the visitorId. The visitorId is returned in the response header (x-photoniq-vid) and the JSON body’s response (visitorId).
+<TabItem value="button click" label="Activate on button click">
+
+With this script, Javascript loading occurs at page load time, and triggers a button to create a function. Clicking this button makes a call to the fingerprint server, which returns the `visitorId` in the response header (x-photoniq-vid) and the JSON body’s response (visitorId).
```
Comment (abhisheklmm) [inline docs/photoniq/fingerprinting/get-started-fingerprinting.md]: this does not seem to be right - `triggers a button to create a function`

Comment (maryann-macrometa) [reply docs/photoniq/fingerprinting/get-started-fingerprinting.md]: Okay, does ' With this script, when Javascript loading occurs at page load time, it creates a function tied to a button'? work better

File: docs/photoniq/fingerprinting/benefits.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+The Fingerprint service generates fingerprints that persist even after visitors take actions to avoid tracking. These actions include:
+
+- Using incognito/private browsing modes
+- Clearing cookies
+- Upgrading browser version
+
+### Browser-Side Agent
+
+Fingerprint uses a browser-side agent. This agent (a browser-side JavaScript library) automatically loads, collects, and transmits information about a visitor's browser to the Fingerprint service. Fingerprint processes this information using proprietary algorithms and advanced machine-learning models to identify web visitors. Some data types collected include user agents, screen resolutions, CPU counts, and others.
```
Comment (abhisheklmm) [inline docs/photoniq/fingerprinting/benefits.md]: lets remove the word `automatically` for now, as there is some work needed to send the data right now

File: docs/photoniq/fingerprinting/benefits.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+### Server-Side Detection
+
+Fingerprint also uses server-side detection for its geolocation service to determine the visitor's location, including city, state, and zip code. It determines if users access your website via a VPN and compares the visitor's browser time zone with the source address's time zone and a list of well-known public VPN servers to identify the VPN used.
```
Comment (abhisheklmm) [inline docs/photoniq/fingerprinting/benefits.md]: technically Fingerprinting doesnot have its `geolocation service`. It uses the IP & ASN information though. Dont know if this needs to be rephrased

Comment (maryann-macrometa) [reply docs/photoniq/fingerprinting/benefits.md]: Done

**Pull request merged**